onadata/settings/common.py

```diff
@@ -557,8 +557,6 @@ def configure_logging(logger, **kwargs):
 
 HOSTNAME = socket.gethostname()
 
-CACHE_MIXIN_SECONDS = 60
-
```

Comment:
Could we have a default `CACHE_CONTROL_DIRECTIVES` setting since we are replacing the `CACHE_MIXIN_SECONDS`?
```python
CACHE_CONTROL_DIRECTIVES = {"max_age": 60}
```

Reply:
We maintain the current behaviour but switch to a new way of defining it.